### PR TITLE
Seperated out tests so that we can test each piece of functionality

### PR DESCRIPTION
**Pull request by Nearhan**
#3 

| MySQL type | Go type | JSON type | Done|
| --- | --- | --- | ---|
| text | string | string | YES |
| binary(n) | []byte | base64 string | NO |
| varchar(n) | string | string | NO |
| boolean | bool | bool | NO |
| timestamp(n) | time.Time | [ISO_8601](https://en.wikipedia.org/wiki/ISO_8601) string | NO |
| int | int64 | float64/numeric | Yes |
| bigint | int64 | float64/numeric | No |

### CODE REVIEW

File: mapper.go [hunk 1]
```diff
@@ -30,6 +30,28 @@ type TableSchema struct {
 	PrimaryKey string
 }
 
+// GetScanner returns a a container that can be scanner
+// for the various data types this table contains
+func (ts TableSchema) GetScanner() []interface{} {
```
Comment (anxiousmodernman) [inline mapper.go:35]: Cool. Might be better named `GetScannable()`

File: mapper.go [hunk 2]
```diff
@@ -143,11 +151,15 @@ func QueryTable(db *sqlx.DB, ts TableSchema) ([]map[string]interface{}, error) {
...
 				log.Println("unhandled type")
 			}
+
+			// check to see if there is a priarmy on this key
 			if ts.ColumnNames()[i] == ts.PrimaryKey {
 				// If this column is the PK; assign it again to a special key in our map
 				m["__pk"] = fmt.Sprintf("%v", m[ts.ColumnNames()[i]])
 			}
 
+			//m = setPrimaryKey(m, ts.PrimaryKey)
```
Comment (anxiousmodernman) [inline mapper.go:161]: no commented out code

File: mapper_test.go [hunk 3]
```diff
@@ -6,6 +6,84 @@ import (
 	"testing"
 )
 
+var TableSchemaFixture = TableSchema{
+	"test",
+	"users",
+	UserColumnSchemaFixture,
```
Comment (anxiousmodernman) [inline mapper_test.go:12]: This is easier to read.

**Review by anxiousmodernman — CHANGES_REQUESTED** [review]
A few recommendations. Let's get the tests to pass on CI.

File: mapper.go (hunk 2 above)
Comment (Nearhan) [reply mapper.go:161]: yeah i need to add that function just mapping out in my head how we should set Primary key. That will remove the check we do on line 156.

Basically check if a column is the Primary key, if not then generate the Guid as you mentioned in some other thing.

**Review by anxiousmodernman — APPROVED** [review]

**Pull request merged**